runtime/lua/vim/snippet.lua

```diff
@@ -446,14 +446,18 @@ function M.expand(input)
       base_indent = base_indent .. (snippet_lines[#snippet_lines]:match('(^%s*)%S') or '') --- @type string
     end
 
+    local sw = vim.fn.shiftwidth()
```

Comment:
Nit, but I prefer to keep the full-variable-name convention in this file:
```suggestion
    local shiftwidth = vim.fn.shiftwidth()
```